.env.sample

```diff
@@ -19,3 +19,6 @@ HMRC_INTERFACE_SECRET='TestHMRCInterfaceSecret'
 
 MOCK_AZURE=false
 MOCK_AZURE_PASSWORD='password'
+
+SIDEKIQ_USERNAME='username'
+SIDEKIQ_PASSWORD='password'
```

Comment:
be tempted to call these SIDEKIQ_WEB_UI_... and elsewhere as the are only for the web ui, as oppsed to redis username and password.